[2025-02-15T01:24:34.130+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-15T01:24:34.135+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_climate.extract_data scheduled__2024-02-16T00:00:00+00:00 [queued]>
[2025-02-15T01:24:34.139+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_climate.extract_data scheduled__2024-02-16T00:00:00+00:00 [queued]>
[2025-02-15T01:24:34.139+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-02-15T01:24:34.147+0000] {taskinstance.py:2889} INFO - Executing <Task(DockerOperator): extract_data> on 2024-02-16 00:00:00+00:00
[2025-02-15T01:24:34.150+0000] {standard_task_runner.py:72} INFO - Started process 133 to run task
[2025-02-15T01:24:34.154+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'etl_climate', 'extract_data', 'scheduled__2024-02-16T00:00:00+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/climate_extract_dag.py', '--cfg-path', '/tmp/tmprbaauqvf']
[2025-02-15T01:24:34.157+0000] {standard_task_runner.py:105} INFO - Job 5: Subtask extract_data
[2025-02-15T01:24:34.183+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_climate.extract_data scheduled__2024-02-16T00:00:00+00:00 [running]> on host 4360a910c8ec
[2025-02-15T01:24:34.225+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='etl_climate' AIRFLOW_CTX_TASK_ID='extract_data' AIRFLOW_CTX_EXECUTION_DATE='2024-02-16T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-16T00:00:00+00:00'
[2025-02-15T01:24:34.226+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-02-15T01:24:34.265+0000] {docker.py:379} INFO - Starting docker container from image dbt_docker
[2025-02-15T01:24:34.279+0000] {docker.py:387} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2025-02-15T01:24:36.375+0000] {docker.py:73} INFO - [0m01:24:36  Running with dbt=1.9.2
[2025-02-15T01:24:36.558+0000] {docker.py:73} INFO - [0m01:24:36  Registered adapter: duckdb=1.9.1
[2025-02-15T01:24:36.605+0000] {docker.py:73} INFO - [0m01:24:36  Unable to do partial parsing because saved manifest not found. Starting full parse.
[2025-02-15T01:24:37.674+0000] {docker.py:73} INFO - [0m01:24:37  Found 2 models, 1 seed, 28 data tests, 424 macros
[2025-02-15T01:24:37.675+0000] {docker.py:73} INFO - [0m01:24:37
[2025-02-15T01:24:37.676+0000] {docker.py:73} INFO - [0m01:24:37  Concurrency: 1 threads (target='dev')
[2025-02-15T01:24:37.676+0000] {docker.py:73} INFO - [0m01:24:37
[2025-02-15T01:24:39.192+0000] {docker.py:73} INFO - [0m01:24:39  1 of 1 START sql table model main.open_meteo_raw_data .......................... [RUN]
[2025-02-15T01:25:25.335+0000] {docker.py:73} INFO - [0m01:25:25  1 of 1 OK created sql table model main.open_meteo_raw_data ..................... [[32mOK[0m in 46.14s]
[2025-02-15T01:25:25.376+0000] {docker.py:73} INFO - [0m01:25:25
[2025-02-15T01:25:25.377+0000] {docker.py:73} INFO - [0m01:25:25  Finished running 1 table model in 0 hours 0 minutes and 47.70 seconds (47.70s).
[2025-02-15T01:25:25.450+0000] {docker.py:73} INFO - [0m01:25:25
[2025-02-15T01:25:25.451+0000] {docker.py:73} INFO - [0m01:25:25  [32mCompleted successfully[0m
[2025-02-15T01:25:25.451+0000] {docker.py:73} INFO - [0m01:25:25
[2025-02-15T01:25:25.452+0000] {docker.py:73} INFO - [0m01:25:25  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-15T01:25:25.935+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-02-15T01:25:25.936+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=etl_climate, task_id=extract_data, run_id=scheduled__2024-02-16T00:00:00+00:00, execution_date=20240216T000000, start_date=20250215T012434, end_date=20250215T012525
[2025-02-15T01:25:25.963+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-15T01:25:25.979+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-15T01:25:25.979+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
